src/main/java/net/mcreator/ui/workspace/resources/WorkspacePanelTextures.java

```diff
@@ -224,6 +227,16 @@ private void duplicateSelectedFile() {
 		}
 	}
 
+	private void replaceSelectedFile() {
+		File file = listGroup.getSelectedItem();
+		File newTexture = FileDialogs.getOpenDialog(workspacePanel.getMCreator(), new String[] { ".png" });

```

Comment:
If no item is selected, the dialog should not be shown

Reply:
True